components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FetchBuildValidationDataCommand.java

```diff
@@ -45,8 +45,8 @@ public static class ExitCode {
     @Option(names = {"-m", "--mapping-file"}, description = "Specifies a mapping file that configures the keys used to fetch important custom values.")
     private Optional<Path> customValueMappingFile;
 
-    @Option(names = {"-p", "--proxy-settings-file"}, description = "Specifies a file that configures HTTP Proxy settings.")
-    private Optional<Path> proxySettingsFile;
+    @Option(names = {"--network-settings-file"}, description = "Specifies a file that configures HTTP Proxy and SSL settings.")
```

Comment:
`...that configures HTTP(S) Proxy and...`

Reply:
✅ Done